eventually/mentor/views.py

```diff
@@ -103,17 +136,34 @@ def post(self, request):
         record = list(MentorStudent.objects.filter(student_id=student.id, topic_id=topic.id))
```

Comment:
don't use objects in view

Reply:
add method topic_student_belonging, that returns MentorStudent instance